### PR TITLE
Fix lint warnings #147

### PR DESCRIPTION
**Pull request by joonki34**
Fixed 56 lint warnings out of 72.
Some warnings are just suppressed since they are unable to fix (e.g. ExportedReceiver lint warning).

### CODE REVIEW

**Comment by ilya-murzinov** [comment]
@joonki34, great, will review today!

File: wail-app/src/main/java/com/artemzin/android/wail/storage/db/IgnoredPlayersDBHelper.java [hunk 1]
```diff
@@ -70,6 +70,8 @@ public synchronized long add(String packageName) {
             }
         }
 
+        cursor.close();
```
Comment (artem-zinnatullin) [inline wail-app/src/main/java/com/artemzin/android/wail/storage/db/IgnoredPlayersDBHelper.java]: Wow, thanks for that!

File: wail-app/src/main/java/com/artemzin/android/wail/ui/activity/MainActivity.java [hunk 2]
```diff
@@ -142,7 +142,7 @@ public View getView(int position, View view, ViewGroup parent) {
 
                 if (rowView == null) {
                     LayoutInflater inflater = getLayoutInflater();
-                    rowView = inflater.inflate(R.layout.activity_main_drawer_item_layout, null, true);
+                    rowView = inflater.inflate(R.layout.activity_main_drawer_item_layout, parent, false);
```
Comment (artem-zinnatullin) [inline wail-app/src/main/java/com/artemzin/android/wail/ui/activity/MainActivity.java]: Fail…

**Comment by artem-zinnatullin** [comment]
Not sure about RTL UI now, but LGTM!

**Comment by ilya-murzinov** [comment]
@artem-zinnatullin, please merge it if it's OK.
I'll rebase my #150 after.

**Comment by artem-zinnatullin** [comment]
@joonki34 can you please post screenshot with RTL changes that you've made here?

**Comment by joonki34** [comment]
@artem-zinnatullin Oh, I just fixed lint warnings regarding this issue. Since I don't know any of rtl languages, I am not sure that the fixes would work on RTL UI so I can't post screenshots.  Maybe I should set android:supportsRtl as false in AndroidManifest.xml for now.

File: wail-app/src/main/java/com/artemzin/android/wail/ui/activity/MainActivity.java (hunk 2 above)
Comment (joonki34) [inline wail-app/src/main/java/com/artemzin/android/wail/ui/activity/MainActivity.java]: @artem-zinnatullin Can you show me the error message you've got? This works on my device and emulator.

**Comment by ilya-murzinov** [comment]
@joonki34, @artem-zinnatullin, I'm sorry, but what are you talking about? What is RLT or RTL UI? :)

**Comment by ilya-murzinov** [comment]
@joonki34, @artem-zinnatullin, nevermind, I get it :)

**Comment by artem-zinnatullin** [comment]
:) Right-To-Left, some Arabic (or Arabian) languages use RTL style.
16 июня 2015 г. 10:40 AM пользователь "Ilya Murzinov" <
notifications@github.com> написал:

**Comment by ilya-murzinov** [comment]
@artem-zinnatullin, I've tested this, seems OK to me.
@joonki34, thank you for you work!

**Comment by artem-zinnatullin** [comment]
Great! Thanks @joonki34 and @ilya-murzinov

**Pull request merged**